xcode/Subconscious/Shared/Components/Common/EntryList/EntryListEmptyView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  EntryListEmptyView.swift
+//  Subconscious (iOS)
+//
+//  Created by Gordon Brander on 6/28/23.
+//
+
+import SwiftUI
+
+struct EntryListEmptyView: View {
+    var onRefresh: () -> Void
+
+    var body: some View {
+        GeometryReader { geom in
+            ScrollView {
+                VStack {
+                    Spacer()
+                    HStack {
+                        Spacer()
+                        VStack(spacing: AppTheme.unit * 6) {
+                            Image(systemName: "sparkles")
+                                .font(.system(size: 64))
+                            Text("Your Subconscious is empty")
+                            VStack(spacing: AppTheme.unit) {
+                                Text(
+                                    "If your mind is empty, it is always ready for anything, it is open to everything. In the beginner's mind there are many possibilities, but in the expert's mind there are few."
+                                )
+                                .italic()
+                                Text(
+                                    "Shunryū Suzuki"
+                                )
+                            }
+                            .frame(width: 240)
+                            .font(.caption)
+                        }
+                        Spacer()
+                    }
+                    Spacer()
+                }
+                .multilineTextAlignment(.center)
+                .padding()
+                .frame(minHeight: geom.size.height)
```

Comment:
Assuming this is to ensure the user can drag from anywhere on the screen?

Reply:
By default, views in scrollviews shrink vertically to a height defined by their contents.

We set an explicit height for the view so that the quote within will be vertically centered.